packages/showcases/src/components/FlowDiagram.tsx

```diff
@@ -17,46 +21,85 @@ const getFlowChartAsset = (showcase: string) => {
+        background: "black"
+    },
+    content: {
+        top: '50%',
+        left: '50%',
+        right: 'auto',
+        bottom: 'auto',
+        marginRight: '-50%',
+        transform: 'translate(-50%, -50%)',
+    },
+};
 
 
 export const FlowDiagram = () => {
     const [show, setShow] = useState(true);
     const {asPath} = useRouter()
     const path = asPath.replace('/', '')
+    const [modalIsOpen, setIsOpen] = React.useState(false);
+
 
     function toggleShow() {
         setShow((prev) => !prev);
     }
 
+
+    function openModal() {
+        setIsOpen(true);
+    }
+
+    const [y, setY] = useState(0);
+
     return (
-        <div className={"fixed bottom-0 w-full bg-gray-100 p-4 py-2 " + (!show && "bg-opacity-0")}>
-            <div className={"items-center flex fex-row " + (!show && "flex-row-reverse ")}>
-                {show && <a
-                    href="https://get.jina.ai"
-                    rel="noreferrer"
-                    target="_blank"
-                    className="md:mt-2 flex flor-row items-center cursor-pointer hover:bg-gray-200 md:p-2 rounded-md tranistion-all duration-200">
-                    <img src={github} alt="github"/>
-                    <span className="ml-2">Jina on Github</span>
-                </a>}
-                {show && <div className="md:text-lg md:mt-2 flex-1 font-medium text-center">How it works</div>}
-                <ChevronUpIcon
-                    onClick={toggleShow}
-                    className={`md:mt-2 h-10 p-2 rounded-full hover:bg-gray-200 cursor-pointer transition-all duration-200 transform ${
-                        show ? "rotate-180" : "rotate-0"
-                    }`}
-                />
+        <div
+            className={"fixed bottom-0 w-full bg-gray-100 p-4 py-2 rounded-xl cursor-pointer"}
+            onClick={() => toggleShow()}
+        >
+            <Modal
+                isOpen={modalIsOpen}
+                onRequestClose={() => setIsOpen(false)}
+                style={customStyles}
+                contentLabel="Example Modal"
+            >
+                <img src={getFlowChartAsset(path)} alt="flow"/>
+
+            </Modal>
+            <div className="flex justify-center mt-3">
+                <img src={DragAndDropRec} alt={"dragAndDropRec"}/>
             </div>
+            <div className="md:text-lg md:mt-2 flex-1 font-medium mb-3">Flow diagram</div>
+
             <div
                 className={`overflow-hidden transition-all duration-200 ease-in-out ${
                     show ? "max-h-screen" : "max-h-0"
                 }`}
             >
-                <div className="flex justify-center pb-8 bg-gray-100 rounded mt-4 overflow-y-auto">
+                <div className="flex justify-center">
+                    <div className="flex text-gray-600">
+                        <img src={IconPlusSearch} alt="IconPlus"/>
+                        <span className="ml-1">
+                         Click to check full image
```

Comment:
I think we should have `setIsOpen` triggered when you click on this text as well. In the UI, I expected it to open on this.